### PR TITLE
restic: update to 0.13.1

### PR DESCRIPTION
**Pull request by Torxgewinde**
Signed-off-by: Tom Stöveken tom@naaa.de

Tested lantiq/xrx200

Maintainer: unchanged, it was Markus Weippert markus@gekmihesg.de
Compile tested: SDK for OpenWrt 21.02.3
Run tested: xRX200 rev 1.2, AVM FRITZ!Box 7360 V2, OpenWrt 21.02.3

Description:
    Updated to version 0.13.1

### CODE REVIEW

File: utils/restic/Makefile [hunk 1]
```diff
@@ -1,12 +1,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=restic
-PKG_VERSION:=0.9.6
-PKG_RELEASE:=2
+PKG_VERSION:=0.13.1
+PKG_RELEASE:=3
```
Comment (jefferyto) [inline utils/restic/Makefile:5]: Please reset `PKG_RELEASE` to `1` or set to `$(AUTORELEASE)`.

Comment (Torxgewinde) [reply utils/restic/Makefile]: will do, I am still struggling with git

**Comment by Torxgewinde** [comment]
Guess I need to start a new PR again

**Comment by Torxgewinde** [comment]
Ok, will do. Sorry for the mess-up. Unfortunately I haven't practiced git before...

**Comment by jefferyto** [comment]
It's okay, we all started as beginners :smile:

**Pull request closed without merging**